graspy/simulations/simulations.py

```diff
@@ -268,6 +268,21 @@ def sbm(n, p, directed=False, loops=False, wt=1, wtargs=None):
         if Wt is an object, Wtargs corresponds to the trailing arguments
         to pass to the weight function. If Wt is an array-like, Wtargs[i, j] 
         corresponds to trailing arguments to pass to Wt[i, j].
+    dc: function or array-like, shape (n_vertices)
+        if dc is a function, it should generate a random number to be used
+        as a weight to create a heterogenous degree distribution. A weight 
```

Comment:
Only other comment is whether scalar dc's should just be normalized if they don't sum to 1, just like if they hade been generated by a function. I'm not saying they should, I just think it's something to consider.

Reply:
I had thought about that too. I just committed a revision to normalize it by default but throw a warning too if it does so. I think it's simple enough for the input to be normalized and not having some sort of check opens room for error during simulations.